### PR TITLE
[fr] confusion dans une expression rule

### PR DESCRIPTION
**Pull request by Rebecca-auque**
Confusion dans une expression.

### CODE REVIEW

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 1]
```diff
@@ -37493,6 +37493,110 @@ though, and has since been slightly modified)-->
...
+                  <token>suite</token>
+                  <token regexp="yes">à|entières?</token>
+              </antipattern>
+              <antipattern>
+                  <token regexp="yes" skip="-1">pas|jamais|plus</token>
+                  <token>de</token>
+                  <token>suite</token>
+              </antipattern>
+              <antipattern>
+                  <token regexp="yes">toute?s?|ainsi|pas|jamais|plus|formes?|beaucoup|faute|défaites?</token>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:37534]: "trois fois de suite", is that a plausible expression? (if so, add "fois" to AP)

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml [hunk 2]
```diff
@@ -37493,6 +37493,110 @@ though, and has since been slightly modified)-->
...
+              <message>Cette expression familière est fautive.</message>
+              <suggestion>détente</suggestion>
+              <example correction="détente">Il a appuyé sur la <marker>gâchette</marker>.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <token>s'</token>
+                  <token inflected="yes">accaparer</token>
+              </pattern>
+              <message>Une erreur semble être détectée car \2 n'est pas un verbe pronominal.</message>
```
Comment (udomai) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:37586]: Don't forget to put \2 in 'single quotes'.

Message: ...un verbe **réflexif** ? Or are they commonly called "verbes pronominaux"?

**Review by udomai — CHANGES_REQUESTED** [review]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 1 above)
Comment (Rebecca-auque) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:37534]: Yes, that is possible so I added the AP.

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml (hunk 2 above)
Comment (Rebecca-auque) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml:37586]: They are commonly called "verbes pronominaux".

**Comment by udomai** [comment]
manually merged: 8fafe053e277f884c196981ac2ee82832dd0212f

**Pull request closed without merging**